Update documentation for Kubeapps 1.8 (#1475)
kubeapps/kubeapps: README.md

Use the Helm chart to install the latest version of Kubeapps: 

> ***Note:*** Kubeapps supports both Helm v2 and v3. For detailed instructions on how to install and use Kubeapps follow the [Getting Started Guide](docs/user/getting-started.md).

Reviewer: ```suggestion
***Note:*** Kubeapps supports both Helm 2 and Helm 3. For detailed instructions on how to install and use Kubeapps follow the [Getting Started Guide](docs/user/getting-started.md).
```
This [seems to be the official style](https://helm.sh/blog/helm-3-released/). Hence/also, I think it would improve `Ctrl+F`-ability?  WDYT?

```bash
helm repo add bitnami https://charts.bitnami.com/bitnami

> # Helm v2 command

Reviewer: ```suggestion
# Helm 2 command
```

helm install --name kubeapps --namespace kubeapps bitnami/kubeapps

> # Helm v3 command

Reviewer: ```suggestion
# Helm 3 command
```

kubectl create namespace kubeapps

> helm install --name kubeapps --namespace kubeapps bitnami/kubeapps

Reviewer 2: Looks like a copy-n-pasto, as `--name` isn't supported on the helm3 cli:
```
$ helm install --name kubeapps --namespace kubeapps bitnami/kubeapps
Error: unknown flag: --name
```

Also, we need the `--useHelm3` flag. (EDIT: exactly like you have in the chart readme below)
